test/TestAssets/TestProjects/VanillaWasm/VanillaWasm.csproj

```diff
@@ -2,5 +2,11 @@
   <PropertyGroup>
     <TargetFramework>$(CurrentTargetFramework)</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <WasmFingerprintDotnetJs>true</WasmFingerprintDotnetJs>
+    <WriteImportMapToHtml>true</WriteImportMapToHtml>
   </PropertyGroup>
+  <ItemGroup>
+    <StaticWebAssetFingerprintPattern Include="Js" Pattern="*.js" Expression="#[.{fingerprint}]!" />
```

Comment:
How does it find the .html file ? Would it manipulate all of them ?

`Include="Js"` - what are other possible values ?

Reply:
Which code reads `Expression` ?

Reply:
> Include="Js" - what are other possible values ?

The identity irrelevant, the import parts are `Pattern` & `Expression`